### PR TITLE
Remove deprecated listener events

### PR DESCRIPTION
**Pull request by jderusse**
Fixes https://github.com/getsentry/sentry-symfony/pull/266#issuecomment-558170085

### CODE REVIEW

**Comment by Jean85** [comment]
This should be done in #277; but I see that you're covering more cases, I need to expand my E2E tests!

File: src/EventListener/RequestListener.php [hunk 1]
```diff
@@ -119,3 +121,15 @@ private function getUserData($user): array
         return [];
     }
 }
+
+if (\class_exists(ResponseEvent::class)) {
```
Comment (jderusse) [inline src/EventListener/RequestListener.php]: Should be moved outside the bundle. Let me time to fix that

Comment (jderusse) [reply src/EventListener/RequestListener.php]: nvm, typehint does not trigger autoload... I've to include them in the Listener

**Comment by jderusse** [comment]
@Jean85 sorry, I didn't checked your PR. Fill free to cherry-pick or copy/paste whatever you wan't in mine PR :-)

**Comment by Jean85** [comment]
I think I'll cherry pick your commits, thanks! I just pushed new tests with 2712152

**Comment by Jean85** [comment]
Solved in #266

**Pull request closed without merging**